app/templates/frameworks/_agreement_returned_legal.html

```diff
@@ -0,0 +1,106 @@
+        {% endcall %}
+      {% endcall %}
+      {% call summary.row() %}
+        {{ summary.field_name('Submitted by') }}
+        {% call summary.field() %}
+          <p>{{ supplier_framework.agreementDetails.uploaderUserName }}</p>
+          <p>{{ supplier_framework.agreementDetails.uploaderUserEmail }}</p>
+          <p>{{ supplier_framework.agreementReturnedAt|datetimeformat }}</p>
+        {% endcall %}
+      {% endcall %}
+      {% call summary.row() %}
+        {{ summary.field_name('Countersignature') }}
+        {{ summary.field_name('Waiting for CCS to countersign') }}
+      {% endcall %}
+    {% endcall %}
+  {% endif %}
+
+    <div class="dmspeak">
+    {% if framework_urls.framework_agreement_url %}
+      {# newer frameworks should have the final agreement published as a web page #}
+      <p>
+        <a href="{{ framework_urls.framework_agreement_url }}">
+          Read the standard framework agreement
+        </a>
+      </p>
+    {% elif communications_files.final_agreement.last_modified %}
+      {# older frameworks will still have to point to a file that came from the old supplier pack #}
+      <p>
+        <a href="{{ url_for('.download_supplier_file', filepath=communications_files.final_agreement.filename, framework_slug=framework.slug) }}">
+          Download the standard framework agreement
+        </a>
+      </p>
+    {% endif %}
+      <p>
+        <a href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name=signed_agreement_document_name) }}" target="_blank">Download your ‘original’ framework agreement signature page</a>
+        <br/>
+        The agreement signed by your company
+      </p>
+    {% if countersigned_agreement_file %}
+      <p>
+        <a href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name=countersigned_agreement_file) }}" target="_blank">Download your ‘counterpart’ framework agreement signature page</a>
+        <br/>
+        The agreement signed by the Crown Commercial Service
+      </p>
+    {% endif %}
+    {% if 'CONTRACT_VARIATION' is active_feature %}
+      {% for variation in framework.variations %}
+      <p>
+        <a href="{{ url_for('.view_contract_variation', framework_slug=framework.slug, variation_slug=variation) }}">
+        {% if framework.variations[variation].get('countersignedAt') and supplier_framework.agreedVariations.get(variation).agreedAt %}
+          View the signed contract variation
+        {% else %}
+          Read the proposed contract variation
+        {% endif %}
+        </a>
+      </p>
+      {% endfor %}
+    {% endif %}
+    {% if communications_files.call_off.last_modified %}
+      <p>
+        <a href="{{ url_for('.download_supplier_file', filepath=communications_files.call_off.filename, framework_slug=framework.slug) }}">Download the call-off contract template</a>
+      </p>
+    {% endif %}
+    {% if not supplier_framework.countersigned %}
```

Comment:
...and @laurenceberry says this is the correct behaviour - they need a reassurance paragraph pre-countersigning but not otherwise.